google-cloud-spanner/lib/google/cloud/spanner/admin/database/v1/doc/google/iam/v1/policy.rb

```diff
@@ -121,19 +121,7 @@ class PolicyDelta; end
       #     A single identity requesting access for a Cloud Platform resource.
       #     Follows the same format of Binding.members.
       #     Required
-      class BindingDelta
```

Comment:
Is this removal still expected? See https://github.com/GoogleCloudPlatform/google-cloud-ruby/pull/1727#discussion_r139265901

Reply:
I suspect this is a regression, but would like to decouple it from this refresh. I filed https://github.com/googleapis/toolkit/issues/1678 to investigate.

Reply:
Should this block approval and merging of this PR?

Reply:
I manually reverted this. (I grepped through the diff, and this seems to be the only occurrence where a module within a doc file has gone missing.)